components/scream/tests/uncoupled/homme/input.yaml

```diff
@@ -4,12 +4,15 @@ Debug:
   Atmosphere DAG Verbosity Level: 5
 
 Initial Conditions:
-  Initial Conditions File: homme_standalone_ne4np4.nc
-  # Set X_prev = X
-  T_mid_prev: T_mid
-  horiz_winds_prev: horiz_winds
-  w_int: 0.0 # Start from zero vert velocity
-  w_int_prev: 0.0
+  Physics GLL:
+    Initial Conditions File: homme_standalone_ne4np4.nc
```

Comment:
What about just `File`? It's already in the `Initial Conditions` section/block

Reply:
I agree.